src/js/components/Button/StyledButtonKind.js

```diff
@@ -32,7 +32,8 @@ const fontStyle = props => {
   `;
 };
 
-const padFromTheme = (size, theme) => {
+const padFromTheme = (sizeProp, theme) => {
```

Comment:
FYI, I think you could have just done: `(size = 'medium', theme)`

Reply:
Good reminder, updating now.